src/api/ecom-api.tsx

```diff
@@ -107,4 +117,4 @@ function getEcomApi(wixClient: ReturnType<typeof getWixClient>) {
     };
 }
 
-export const ecomApi = getEcomApi(getWixClient());
+export const getEcomApi = () => createEcomApi(getWixClient());
```

Comment:
let's maybe unite `getEcomApi` and `createEcomApi`?

Reply:
fixed